docs/Reference/API-Methods.md

```diff
@@ -194,6 +194,56 @@ Generates cached log bloom indexes for blocks. APIs such as [`eth_getLogs`](#eth
         }
         ```
 
+### `admin_logsRemoveCache`
+
+Removes cache files for the specified range of blocks.
+
+#### Parameters
+
+`fromBlock` - Integer representing a block number or one of the string tags `latest`,
+`earliest`, or `pending`, as described in
+[Block Parameter](../HowTo/Interact/APIs/Using-JSON-RPC-API.md#block-parameter).
+
+`toBlock` - Integer representing a block number or one of the string tags `latest`,
+`earliest`, or `pending`, as described in
+[Block Parameter](../HowTo/Interact/APIs/Using-JSON-RPC-API.md#block-parameter).
+
+If you specify:
+
+* No parameters, the call removes cache files for all blocks.
+* Only the first parameter, the call removes cache files for the specified block.
```

Comment:
Name the parameters in addition to giving the position.
Explain how to skip a parameter. Does it require an empty "" value, a null, something else?